InvenTree/InvenTree/tasks.py

```diff
@@ -306,4 +307,35 @@ def send_email(subject, body, recipients, from_email=None):
         from_email,
         recipients,
         fail_silently=False,
+        html_message=html_message
     )
+
+
+def notify_low_stock(stock_item):
```

Comment:
This function should be moved to `InvenTree/part/tasks.py` as it relates to the part model.